.markdownlint.jsonc

```
@@ -208,7 +208,7 @@
       {
         "name": "un-closed-md-link",
         "message": "Missing closing bracket ')'",
-        "searchPattern": "/(\\[[^\\]]*?\\]\\(([^\\)\\n]|\\([^\\)\\n]\\)|\\s[\"'])+?)(\\n|\\s|[,:][\\s\\n])/gm",
+        "searchPattern": "/\\[[^\\]]*?\\]\\(([^ )\n\",:])+?([\n,:\"]| \\w| [\"'][^\"']+?[\"'][^)])/gm",
```

```suggestion
        "searchPattern": "/\\[.*?\\]\\(([^ )\n\",:])+([\n\",:]| [^\"']| [\"'].+?[\"'][^)])/gm",
```

Slightly simpler (`[x][^x]+?[x]` is equivalent to `[x].+?[x]`)